app/views/shared/_title.html.haml

```diff
@@ -1,6 +1,6 @@
-%section.hero.opaque
+%section.stripe.reverse
   .row
```

Comment:
I haven't been able to double check the css so you may need to ignore me, but these deleted id's don't have any related styles that need deleting do they?

Reply:
I have fixed this in e95457f below